### PR TITLE
Extract ToModelProto for GraphViewer and Function

### PR DESCRIPTION
**Pull request by daquexian**
**Description**:
As requested in https://github.com/microsoft/onnxruntime/pull/1220#discussion_r297743898, extract `ToModelProto` method for `onnxruntime::GraphViewer` and `onnxruntime::Function`

**Motivation and Context**
- Why is this change required? What problem does it solve?
Avoid duplicate codes in execution providers
- If it fixes an open issue, please link to the issue here.
https://github.com/microsoft/onnxruntime/pull/1220#discussion_r297743898

### CODE REVIEW

File: onnxruntime/core/graph/graph_utils.cc [hunk 1]
```diff
@@ -448,6 +448,22 @@ size_t RemoveNodeOutputEdges(Graph& graph, Node& node) {
   return output_edges.size();
 }
 
+ONNX_NAMESPACE::ModelProto GetModelProto(const onnxruntime::GraphViewer& graph) {
+  ONNX_NAMESPACE::ModelProto model_proto;
+  *model_proto.mutable_graph() = graph.ToGraphProto();
+  model_proto.set_ir_version(ONNX_NAMESPACE::Version::IR_VERSION);
+  return model_proto;
```
Comment (linkerzhang) [inline onnxruntime/core/graph/graph_utils.cc]: I think there're still other fields in ModelProto need to be set, for example, opset_import.

**Comment by daquexian** [comment]
@linkerzhang I have updated the branch :)

File: onnxruntime/core/graph/graph_utils.cc (hunk 1 above)
Comment (daquexian) [reply onnxruntime/core/graph/graph_utils.cc]: Thanks! I have added it :)

**Pull request closed without merging**